prow/plugins.yaml

```diff
@@ -21,6 +21,7 @@ kubernetes/kubernetes:
 kubernetes/test-infra:
 - trigger
 - config-updater
+- lint
```

Comment:
Done. In the future we will probably want to add more linters, but those should likely be added as a separate service in prow. We don't want to be installing too much stuff into the hook image.